network/metadata.csv

```diff
@@ -47,6 +47,8 @@ system.net.packets_in.count,gauge,,packet,second,The number of packets of data r
 system.net.packets_in.error,gauge,,error,second,The number of packet receive errors detected by the device driver.,-1,system,pkts in err
 system.net.packets_out.count,gauge,,packet,second,The number of packets of data transmitted by the interface.,0,system,packets out
 system.net.packets_out.error,gauge,,error,second,The number of packet transmit errors detected by the device driver.,-1,system,pkts out err
+system.net.tcp.failed_retransmits,gauge,,packet,,Number of TCP packets that failed to be retransmitted,0,system,failed retransmit
+system.net.tcp.failed_retransmits.count,count,,packet,,Number of TCP packets that failed to be retransmitted,0,system,failed retransmit count
```

Comment:
Are these still `(Linux only)`? And we should state that the metric 51 is the monotonic count, the descriptions are the same and will confuse users

Reply:
I'm not sure if they are still linux only?
I can update line 51

Reply:
Yes, looks like only BSD submits this metric https://github.com/DataDog/integrations-core/blob/master/network/datadog_checks/network/network.py#L497